docs/book/src/quick-start.md

```diff
@@ -193,6 +196,11 @@ privileges or be logged in as admin. See [Prerequisites for using Kubernetes RBA
 
 </aside> 
 
-Now, follow up the tutorial for you understanding better how it works and check an full example project working. 
+## Next Step 
+
+Now, follow up the CronJob tutorial[cronjob-tutorial] for you understanding better how it works and check a demo example project working on. 
```

Comment:
"for you understanding better" ➡️ "to better understand"
"check" ➡️ "try".
delete "working on".

Reply:
Done